<launch>

<<<<<<< HEAD
  <!-- <node pkg="rosserial_python" type="serial_node.py" name="ros_arduino_imu" output="screen" respawn="true">
    <param name="~port" value="/dev/ttyACM0" />
    <param name="~baud" value="115200" />
  </node>  -->
<!-- 
  <node pkg="imu_f" type="imu_bridge_node" name="imu_bridge" output="screen" respawn="false">
   <rosparam>
   imu/gyroscope_bias: {x: 0, y: 0, z: 0}
   imu/accelerometer_bias: {x: 0, y: 0, z: 0}
   </rosparam>
   <param name="imu/perform_calibration" value="false" />
   <param name="imu/use_mag_msg" value="true" />
=======
  <node pkg="imu_f" type="imu_bridge" name="imu_bridge" respawn="false">
    <rosparam>
      imu/gyroscope_bias: {x: -0.016816, y: -0.043712, z: 0.002788}
      imu/accelerometer_bias: {x: 0.605000, y: -0.246953, z: -0.220859}
    </rosparam>
    <param name="imu/perform_calibration" value="false" />
    <param name="imu/use_mag_msg" value="true" />
>>>>>>> 0b73d1b3c47b9dab4bf99dfefdfebef4debf2545

  </node> -->

<<<<<<< HEAD
	<node pkg="imu_f" type="odom_encoder" name="odom_encoder" output="screen">
    <param name="odom_linear_scale_correction" type="double" value="1.0" />
    <param name="odom_angular_scale_correction" type="double" value="1.0" />
=======
  <node pkg="imu_f" type="odom_encoder" name="odom_encoder" output="screen">
    <param name="odom_linear_scale_correction" type="double" value="0.615" />
    <param name="odom_angular_scale_correction" type="double" value="0.79" />
>>>>>>> 0b73d1b3c47b9dab4bf99dfefdfebef4debf2545
  </node>

  <node pkg="imu_f" type="imu_vel" name="imu_vel" output="screen"/>

<<<<<<< HEAD
  <!-- <node pkg="imu_complementary_filter" type="complementary_filter_node"  name="complementary_filter_gain_node" output="screen">
    <param name="fixed_frame" value="odom"  />
=======
  <node pkg="imu_complementary_filter" type="complementary_filter_node" name="complementary_filter_gain_node" output="screen">
    <param name="fixed_frame" value="odom" />
>>>>>>> 0b73d1b3c47b9dab4bf99dfefdfebef4debf2545
    <param name="use_mag" value="false"/>
  </node> -->


</launch>
